src/regions/ca/info/faq/15_metro.fr.md

```diff
@@ -1,8 +1,8 @@
 ## Dois-je arrêter de prendre les transports publics?
 
-Veuillez consulter les directives de votre gouvernement provincial ou de votre ville. Si vous utilisez les transports publics, veillez à prendre des mesures préventives : lavez-vous soigneusement les mains après, évitez de vous toucher le visage, etc.
+Veuillez consulter les directives de votre gouvernement provincial ou de votre ville. Si vous utilisez les transports publics, veillez à prendre des mesures préventives: lavez-vous soigneusement les mains après, évitez de vous toucher le visage, etc.
 
 Plus d'information par ici:
 
 - [Gouvernement du Canada](https://www.canada.ca/fr/sante-publique/services/maladies/2019-nouveau-coronavirus/prevention-risques.html)
-- [Gouvernement du Québec](https://www.quebec.ca/sante/problemes-de-sante/a-z/coronavirus-2019/reponses-questions-coronavirus-covid19/) (sous la rubrique _Transport en commun_ )
+- [Gouvernement du Québec](https://www.quebec.ca/sante/problemes-de-sante/a-z/coronavirus-2019/reponses-questions-coronavirus-covid19/) (sous la rubrique _Transport en commun_)
```

Comment:
this can link directly to the `transport en commun` section: https://www.quebec.ca/sante/problemes-de-sante/a-z/coronavirus-2019/reponses-questions-coronavirus-covid19/#c46821
```suggestion
- [Gouvernement du Québec](https://www.quebec.ca/sante/problemes-de-sante/a-z/coronavirus-2019/reponses-questions-coronavirus-covid19/#c46821) (sous la rubrique _Transport en commun_)
```

Reply:
Très bon point! Pour l'espace avant la parenthèse je pensais l'avoir corrigé. Bien vu!

Reply:
c'était pas pour l'espace, mais pour l'URL qui envoie directement a la bonne section!